pages/partials/instructorInfoPanel.ejs

```diff
@@ -16,20 +16,24 @@
   </div>
 
   <% if (locals.authz_result && (!authz_result.authorized_edit)) { %>
```

Comment:
While trying to test this locally, `authz_result.authorized_edit` was true. It's not clear to me why that should prevent this information from showing? I acknowledge that this is unlikely in practice, I think this is only true if I (the person viewing the question) was also the one that created the instance.

We should consider changing this conditional, but if we don't, what's your recommendation for testing this locally?

Reply:
> While trying to test this locally, authz_result.authorized_edit was true. It's not clear to me why that should prevent this information from showing? I acknowledge that this is unlikely in practice, I think this is only true if I (the person viewing the question) was also the one that created the instance.

That's an interesting point. I didn't play with this conditional, but I believe this will also be the case in the question preview, as well as for all students in a group.

> We should consider changing this conditional, but if we don't, what's your recommendation for testing this locally?

You can use the admin query that generates students for a course, then change your effective user to one of the generated students and create a submission. Alternatively, you can use the tools/generate_submissions.py script instead of changing the effective user, that has the advantage of creating submissions for all questions, but is not really necessary in this context.